Dockerfile

```diff
@@ -170,6 +171,20 @@ RUN  --mount=target=/root/.cache,type=cache \
   CGO_ENABLED=0 go build -o /rootlesskit ./cmd/rootlesskit && \
   file /rootlesskit | grep "statically linked"
 
+# Based on https://github.com/containers/fuse-overlayfs/blob/v0.7.6/Dockerfile.static.ubuntu .
+# We use Ubuntu here because Alpine does not provide an apk package for libfuse3.a .
+FROM docker.io/ubuntu:19.10 AS fuse-overlayfs
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+        git ca-certificates libc6-dev gcc make automake autoconf pkgconf libfuse3-dev
+RUN git clone https://github.com/containers/fuse-overlayfs
+WORKDIR fuse-overlayfs
+ARG FUSEOVERLAYFS_VERSION
+RUN git pull && git checkout ${FUSEOVERLAYFS_VERSION}
+RUN  ./autogen.sh && \
+     LIBS="-ldl" LDFLAGS="-static" ./configure && \
+     make && mkdir /out && cp fuse-overlayfs /out
```

Comment:
Some other static builds in here have `file | grep "statically linked"` for safety

Reply:
updated